### PR TITLE
Ignore errors from delete-source which oftens fails

### PR DESCRIPTION
**Pull request by charleso**
Not sure if other people see this. Let me know if there's a better way. I'm still not actually sure doing the delete-source is solving the problem well enough at the moment.

### CODE REVIEW

**Comment by nhibberd** [comment]
NB: I often see issues with this where it removes my lib/* then re-builds them straight away

File: src/cabal [hunk 1]
```diff
@@ -101,7 +101,8 @@ initialize () {
       source_path=$(echo $source | sed s#${PROJECT_ROOT}/##)
       if ! $(echo "${CABAL_SOURCES}" | grep -q "${source_path}"); then
         echo "${source_path} no longer exists - removing from the sandbox"
-        cabal sandbox delete-source "$source" 1> /dev/null
+        # This will fail if there are any remaining non-existent sources
+        cabal sandbox delete-source "$source" 1> /dev/null | true
```
Comment (markhibberd) [inline src/cabal]: `||` perhaps

**Comment by markhibberd** [comment]
@nhibberd Next time you see that, can you check if it has the PROJECT_ROOT fix I put in a couple of weeks ago? It used to use `pwd` which didn't work if you were in a sub-project dir.

**Comment by markhibberd** [comment]
> I'm still not actually sure doing the delete-source is solving the problem well enough at the moment.

Yeh, I don't think it is helping with out removing the installed package. But, this doesn't make that issue any larger so :+1:

**Pull request merged**